### PR TITLE
feat: Add new file and configuration

### PR DESCRIPTION
**Pull request by amornc**
About this orb:

- use image: loadimpact/k6:latest
- fill 'remote_url' (prometheus server full url without '/api/v1/write') 
- fill 'file' script path
- skip_tls_verify: "true" (default)
- save result to artifacts

prerequisite
- env name 'K6_PROMETHEUS_RW_USERNAME' for prometheus user
- env name 'K6_PROMETHEUS_RW_PASSWORD' for prometheus password
- k6 folder (optional)

### CODE REVIEW

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:359877f147ca46486e1b859248847a378a6b45e8

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:5a80689ac622869138883930c2dc01c3450b7133

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:bddf9ff475c621bdaef38088ec1213e709672848

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:f6acb41f3f2ead9dad75bdc01459adb6b95246e1

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:d542c35958301c5f5d8c12027d21487e06a04b36

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:13821ac65901d9dd5e30722d1da9da9118919726

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:4a2c9977d4a168d9212785ac61c25131df842769

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:007470732babcdb64a06889089b629cc26e524be

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:762c1462ce375df46dd77917f59b878ced9a8c2a

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:ad3f518eb053f52cadd05c8fca61122903c83f21

File: src/commands/k6-prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    type: string
+    description: "Prometheus remote URL"
+  skip_tls_verify:
+    type: boolean
+    default: true
+    description: "INSECURE SKIP TLS VERIFY"
+  native_histograms:
+    type: boolean
+    default: true
+    description: "INSECURE SKIP TLS VERIFY"
```
Comment (puizeabix) [inline src/commands/k6-prometheus.yml:16]: Incorrect description

Comment (puizeabix) [reply src/commands/k6-prometheus.yml:16]: Another question krub, How do we authenticate with prometheus

**Review by puizeabix — CHANGES_REQUESTED** [review]
Please see the comments krub

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:cd89192119184bf24f65a1aff1eb0cf5bb55e4d8

File: src/commands/k6-prometheus.yml (hunk 1 above)
Comment (amornc) [reply src/commands/k6-prometheus.yml]: > Another question krub, How do we authenticate with prometheus

k6 remote write output only supports basic authen krub.
image

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:2a97cd5fc67bfe5d0f432690c53690a150b96d71

File: src/commands/k6-prometheus.yml (hunk 1 above)
Comment (amornc) [reply src/commands/k6-prometheus.yml]: > Incorrect description

Done krub.

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:328aa0e77d2597e8c4bbcb0801fe6d1412ed52de

**Comment by amornc** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=dev:328aa0e77d2597e8c4bbcb0801fe6d1412ed52de

**Comment by amornc** [comment]
Your orb has been published to the CircleCI Orb Registry.
You can view your published orb on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/zeabix-cloud-native/k6-prometheus?version=1.0.0

**Pull request merged**